### PR TITLE
Adding CLEAR-10 benchmark to Avalanche

### PR DESCRIPTION
**Pull request by linzhiqiu**
#964 PR for this feature request

### CODE REVIEW

File: avalanche/benchmarks/classic/clear.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        test_split = 'all'
+    elif evaluation_protocol == 'iid':
+        assert seed in SEED_LIST, "No seed for train/test split"
+        train_split = 'train'
+        test_split = 'test'
+    else:
+        raise NotImplementedError()
+    
+    if feature_type is None:
+        assert isinstance(train_transform, type(None)), "No image transform"
```
Comment (lrzpellegrini) [inline avalanche/benchmarks/classic/clear.py:141]: I haven't tried to run the code, but from the main below it seems that non-None transformations should be allowed when feature_type is None...

**Review by lrzpellegrini — CHANGES_REQUESTED** [review]
Thank you, the PR looks great! Everything seems to be in order apart from a comment I left regarding transformations.

File: avalanche/benchmarks/classic/clear.py (hunk 1 above)
Comment (linzhiqiu) [reply avalanche/benchmarks/classic/clear.py]: You are right, I should remove the assertion statement (it was a typo). My original purpose is to make sure the user specify a transformation for image, and not inputting a transformation for feature tensors. But on a second thought, I should leave this choice to users!

**Comment by linzhiqiu** [comment]
Made the change by removing assertion statements.

**Comment by AntonioCarta** [comment]
@lrzpellegrini if you are satisfied we can merge the PR.

**Review by lrzpellegrini — APPROVED** [review]

**Comment by AntonioCarta** [comment]
Thanks again @linzhiqiu

**Pull request merged**